templates/components/badgelist.jsx

```diff
@@ -0,0 +1,39 @@
+var React = require('react'),
+    Badge = require('./badge.jsx'),
+    $ = require('jquery');
+
+var BadgeList = React.createClass({
+  loadUsersFromServer: function() {
+    $.ajax({
```

Comment:
If you are only using jquery for ajax I'd suggest using **fetch** library (I used that on id.wb.org) or use supwragent instead.